src/dapla_pseudo/__init__.py

```diff
@@ -20,11 +20,9 @@
 # Ref: https://github.com/python-poetry/poetry/issues/144#issuecomment-1488038660
 __version__ = version("dapla_toolbelt_pseudo")
 
-from dapla_pseudo.v1.api_models import Field
-from dapla_pseudo.v1.api_models import PseudoKeyset
-from dapla_pseudo.v1.api_models import PseudoRule
```

Comment:
Can you add this again please? I think `PseudoRule` should be easily accessible, ref this use-case: https://github.com/statisticsnorway/dapla-toolbelt-pseudo?tab=readme-ov-file#pseudonymize-using-custom-rules

Reply:
Oops, I didn't intend to remove that. Added back